cmd/nobled/main.go

```diff
@@ -12,7 +12,7 @@ import (
 func main() {
 	rootCmd, _ := cmd.NewRootCmd(
 		"noble",
```

Comment:
While you're touching this section of the codebase, does it make sense to pass the `Name` in here instead of hardcoding the string?

Reply:
Sure does.
Done!